### DISCUSSION TITLE
Idea: Keyboard shortcut for routes filtering

### DISCUSSION THREAD

**Discussion by sotcha** [mockoon/mockoon#449] [Ideas] [answered]
Well, I found extremely useful the new routes filtering feature. 
- What about if a shortcut exists which focuses immediately on the filter field? Something like the Teams client 

<img src="https://user-images.githubusercontent.com/1183265/114689348-49f05900-9d1e-11eb-96e1-63620f9cc365.png" width="400">

When the user presses Ctrl+Shift+F focuses on the user filtering field. 

- Also pressing ESC could clear the field

**Comment by 255kb** [comment, marked as answer]
Hi @sotcha this is indeed a good idea. We are finalizing the next release (due next week) and will stop adding more features and enhancements to prioritize testing. 
Could you please open an issue so we can track this for the next iterations? 
Thanks!

**Reply by sotcha** [reply]
Nice @255kb , I will open an issue for  this.

**Reply by sotcha** [reply]
#451 Opened this
